### PR TITLE
Updated to include PATCH /users/:id

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

File: blueprint.md [hunk 1]
```diff
@@ -877,6 +877,37 @@ Retrieve information about specified user.
...
             }
 
+### Update a User [PATCH]
+> This endpoint can only be used with `Basic` authentication.
+>
+> An admin can update any user in his company.
+> 
+> A standard user can only update himself.
+
+Update information about specified user.
```
Comment (merlinND) [inline blueprint.md]: Recall that only the specified keys will be modified?

File: blueprint.md [hunk 2]
```diff
@@ -877,6 +877,37 @@ Retrieve information about specified user.
...
+
+The `is_admin` flag can only be toggled by an admin.
```
Comment (merlinND) [inline blueprint.md]: by an admin of the user's company.

File: blueprint.md [hunk 3]
```diff
@@ -877,6 +877,37 @@ Retrieve information about specified user.
                 "user_url": "/users/52fb7b90c8318c4dc800006c"
             }
 
+### Update a User [PATCH]
+> This endpoint can only be used with `Basic` authentication.
```
Comment (rricard) [inline blueprint.md:881]: FALSE, it's only needed for updating password.

Comment (Neamar) [inline blueprint.md:881]: No, no exceptions. Basic for everyone, with a middleware check

File: blueprint.md [hunk 4]
```diff
@@ -877,6 +877,37 @@ Retrieve information about specified user.
...
++ Request (application/json)
+
+            {
+                "email": "new-email@gmail.com",
+                "name": "New Name",
+                "password": "passwd",
+                "is_admin": true,
+            }
+
++ Response 204
```
Comment (Neamar) [inline blueprint.md]: 200 :)

**Pull request merged**